### PR TITLE
refactor: of create kafka ams checks for enterprise quota

### PR DESCRIPTION
**Pull request by dimakis**
Closes # 

### Verification Steps

1. Do x
2. Do y

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### CODE REVIEW

**Review by jackdelahunt — APPROVED** [review]

File: pkg/shared/remote/DynamicServiceConstants.go [hunk 1]
```diff
@@ -19,4 +19,5 @@ type AmsConfig struct {
 	TrialProductQuotaID         string `json:"trialQuotaProductId"`
 	LongLivedQuotaProductID     string `json:"longLivedQuotaProductId"`
 	ResourceName                string `json:"resourceName"`
+	EnterpriseProductQuotaID    string `json:"RHOSAKCC"`
```
Comment (machi1990) [inline pkg/shared/remote/DynamicServiceConstants.go:22]: `EnterpriseProductQuotaID    string `json:"enterpriseProductQuotaID"``

Comment (machi1990) [reply pkg/shared/remote/DynamicServiceConstants.go:22]: This has to be added in here as well: https://github.com/dimakis/app-services-cli/blob/b0164e5ea4ac899e8b35d3b8a0972833f86e92b6/pkg/shared/remote/service-constants.json#L4

Comment (machi1990) [reply pkg/shared/remote/DynamicServiceConstants.go:22]: `"enterpriseProductQuotaID": "RHOSAKCC"`

Comment (dimakis) [reply pkg/shared/remote/DynamicServiceConstants.go:22]: damn I was missing it in the json file, I was wondering why it wasn't working. thank you 😄

**Pull request merged**